manifests/tftp/netboot.pp

```diff
@@ -43,12 +43,17 @@
         source => "/boot/efi/EFI/${grub_efi_path}/grubx64.efi",
       }
 
-      file { "${root}/grub2/${shim_file}":
+      file { "${root}/grub2/shimx64.efi":
```

Comment:
If you drop this, can you drop the `$shim_file` variable since it's now unused?

Reply:
It's actually used, the source path is different depending on the RHEL version, this still apply:

        source => "/boot/efi/EFI/${grub_efi_path}/${shim_file}",

It's the target what this patch is changing.

Reply:
Right, good point. Isn't `shimx64.efi` available on EL7?

Reply:
Looks like it is:

```
[root@9d32dc6fa0df /]# rpm -qv shim-x64
shim-x64-15-2.el7.centos.x86_64
[root@9d32dc6fa0df /]# rpm -ql shim-x64
/boot/efi/EFI/BOOT/BOOTX64.EFI
/boot/efi/EFI/BOOT/fallback.efi
/boot/efi/EFI/BOOT/fbx64.efi
/boot/efi/EFI/centos/BOOT.CSV
/boot/efi/EFI/centos/BOOTX64.CSV
/boot/efi/EFI/centos/MokManager.efi
/boot/efi/EFI/centos/mmx64.efi
/boot/efi/EFI/centos/shim.efi
/boot/efi/EFI/centos/shimx64-centos.efi
/boot/efi/EFI/centos/shimx64.efi
```

So is the correct thing to do to copy `shimx64.efi`?

Reply:
Yes, honestly I don't remember why we do have this different behavior for different versions:

    f2c580ccd60898d4aa2676249d67c171  /boot/efi/EFI/fedora/shim.efi
    f2c580ccd60898d4aa2676249d67c171  /boot/efi/EFI/fedora/shimx64.efi

Reply:
I think initially the packages only had `shim.efi` but then needed to handle multiple architectures but now ship it for compatibility. I don't know enough of it to know whether we can control that it only uses the architecture specific ones.